meetings/2024-10-02.md

```diff
@@ -0,0 +1,109 @@
+* MdJ: I like that. I think we can also focus on interop between applications. That uses Solid and make those work together. They might not be UCs we imagine. That's what I'd consider outside-in. Starting from real examples.
+* eP: Coming back to LWS UCs re overlap. As I understand hte harter, the client-client is out of scope. There is not going to be domain-specific. Naturally the WG will not fully satisfy all the UCs because there is always the client-client part. That's why I think the UCs should be shared. Ensuring what we have fits into the LWS.
+* RG: There can be generic use cases, I'll point you to Douglas Engelbart in that respect. Use cases don't necessarily have to be domain-specific. We have a generic UC on what knowledge-workshop would be and thats a good light of how software should be designed.
+* MdJ: To me the topic of how different specs are interoperable is interesting. As an app dev, one might use one and dev another. If all those things are incubating all being interop under Solid.
+
+
+### Use cases
+
+URL: https://solid.github.io/authorization-panel/authorization-ucr/
+URL: https://github.com/solid/user-stories
+URL: https://github.com/solid/webid-profile/blob/main/notes/use-cases.md
+URL: https://solid.github.io/notifications-panel/notifications-ucr
+
+* eP: Authorization use case were written in reverse, starting from WAC as a solution plus some know limitations.
+* eP: Instead of making up use cases, we can work with existing communities, preferably open source and document their real world usecases which already have user base. As an example we had a meeting with https://openfoodnetwork.org/
+* eP: We can also try to "eat our own cookings" more. This prototype tries to define some CG meetings based requirements https://github.com/janeirodigital/sai-js/tree/main/examples/plenary#requirements
+* eP: Unfortunately I didn't participate in the TPAC UC meeting.
+* eP: We (CG) has a bunch of work on this already (linked above).
+* eP: We need to have UCR managed in a way to evaluate proposals. Do we do it by simplicity or emotions or something else? ??? objectively. We can have a matrix for the proposals, then we can see best on practice. With e.g., added complexity, we could do with simpler. This is because different people coming with different proposals. Once we have implementation experience it is going to be subjective. We can have some rough way of evaluating the complexity. For me the main point for the UCs is to evaluate all the proposals. We are kind of stuck on what to focus on.
+* RG: I like a lot of the ideas behind on having some process to make a note of it, but at the end of the day we could do the matrices etc but it'll be subjective and no real way of measuring complexity. It is more of art than science.
```

Comment:
```suggestion
* eP: We need to have UCR managed in a way to evaluate proposals. Do we do it by simplicity, or emotions, or something else? ??? objectively. We can have a matrix for the proposals, then we can see best on practice. With, for instance, added complexity, we could do with simpler. This is because different people come with different proposals. Once we have implementation experience, it is going to be subjective. We can have some rough way of evaluating the complexity. For me, the main point for the UCs is to evaluate all the proposals. We are kind of stuck about what to focus on.
* RG: I like a lot of the ideas about having some process to make a note of it, but at the end of the day we could do the matrices, etc., but it'll be subjective and no real way of measuring complexity. It is more of an art than a science.
```